api/v1beta1/common.go

```diff
@@ -117,16 +117,3 @@ type CommonTestStatus struct {
 	// NetworkAttachments status of the deployment pods
 	NetworkAttachments map[string][]string `json:"networkAttachments,omitempty"`
 }
-
-type WorkflowCommonOptions struct {
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// +kubebuilder:validation:optional
-	// +optional
-	// Use with caution! This parameter specifies whether test-operator should spawn test
-	// pods with allowedPrivilegedEscalation: true and the default capabilities on
-	// top of capabilities that are usually needed by the test pods (NET_ADMIN, NET_RAW).
-	// This parameter is deemed insecure but it is needed for certain test-operator
-	// functionalities to work properly (e.g.: extraRPMs in Tempest CR, or certain set
-	// of tobiko tests).
-	Privileged *bool `json:"privileged,omitempty"`
-}
```

Comment:
This should not be part of the `Fix linting` commit (this probably belongs into the `Embed common struct` commit.)

Reply:
moved to the proper commit